src/app_service/service/settings/service.nim

```diff
@@ -47,6 +47,9 @@ type
   SettingProfilePictureArgs* = ref object of Args
     value*: int
 
+  SettingsIncludeWatchOnlyAccountsArgs* = ref object of Args
```

Comment:
@Khushboo-dev-cpp please name this type more general (like one we have in the line 36), e.g. `SettingsBoolValueArgs` that way we can send the same type whenever we need to send `bool` param and the name of signal is used to differ where that signal comes from.